src/govuk/components/radios/radios.js

```diff
@@ -40,18 +39,27 @@ Radios.prototype.setAttributes = function ($input) {
   var inputIsChecked = $input.checked
   $input.setAttribute('aria-expanded', inputIsChecked)
 
-  var $content = this.$module.querySelector('#' + $input.getAttribute('aria-controls'))
+  var $content = document.querySelector('#' + $input.getAttribute('aria-controls'))
```

Comment:
Just realised I've confused myself.

This line is the one that's important that it isnt global, not the others.

https://github.com/alphagov/govuk-frontend/pull/1156

Going to have to figure this out...

Reply:
I think, we can't assume that 'aria-controls' on a radio means that the user has only conditional reveals.

In this case that we fixed, they put their own aria-controls on the radio for something unrelated.

I think we'll need to do something at the initialisation stage to avoid this...

Reply:
We either need to have a way of querying the `$module` in which the given `$input` exists, or query the whole `document`, as `this.$module` will only search within the scope of the module of the _clicked_ radio.

In terms of checking for aria-controls on something unrelated, yesterday you suggested checking for `govuk-radios__conditional` in `$content.classList`, which sounds like a sane defence?

If we do that, then scoping this to the document seems OK to me as we'll always just be syncing the `--hidden` class with the checked state of the input?

Reply:
Yeah I forgot about the idea of looking for that class, that'll be the ticket thanks Ollie.